Makefile

```diff
@@ -15,7 +15,7 @@ SRC_FILES=$(shell find . -name '*.go' | grep -v '_test.go' | grep -v '/tests/')
 TEST_FILES=$(wildcard *_test.go)
 TELEMETRY_TEST_DIR = $(GO_MGMT_PATH)/build/tests/gnmi_server
 TELEMETRY_TEST_BIN = $(TELEMETRY_TEST_DIR)/server.test
-ifeq ($(SONIC_TELEMETRY_READWRITE),y)
+ifeq ($(SONIC_TELEMETRY_WRITABLE),y)
```

Comment:
where and how this flag will be set?

Reply:
This is set at build time, passed as an environmental variable.

Reply:
Thanks Eric. What is the default value of flag during image build?

In the go build of gnmi_server, will we have a control for read only?

Reply:
The default is read-only. Note that this has already been merged, I am just changing the name of the flag as per request. When the SONIC_TELEMETRY_WRITABLE is set, it adds the following go build flag: ` -tags readwrite` to the go build. Again, the default without the flag is read-only. So if you are using go only, you need to add that to the build to get read-write mode.

Reply:
Thanks!